orc8r/cloud/go/obsidian/swagger/doc.go

```diff
@@ -0,0 +1,20 @@
+/*
+ Copyright 2020 The Magma Authors.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree.
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+/*
+Package swagger provides a simple implementation of the per-service spec
```

Comment:
Since we realized Codacy isn't blocking: I actually like including the block indent for multi-line package comments. But I'll leave the decision up to you